examples/meta/generator/targets/java.json

```diff
@@ -6,7 +6,7 @@
         "IncludeGlobalFunctions": false,
         "DependencyListElement": "import org.shogun.$typeName;",
         "DependencyListSeparator": "\n",
-        "ExcludeImport": ["StringCharList", "StringWordList"]
+        "ExcludeImport": ["StringCharList", "StringWordList", "RealVector", "RealMatrix", "BoolMatrix", "ByteMatrix", "WordMatrix", "IntMatrix", "LongIntMatrix", "ShortRealMatrix", "BoolVector", "IntVector", "ByteVector", "LongIntVector", "ShortRealVector", "WordVector"]
```

Comment:
I am not sure if this is the way to do it, but Java tries to import all of these types. However we don’t need them and I deleted them from SWIG (these were the sgcontainer templates)

Reply:
I think I used the Sg type because of the strange api of double matrix. Would prefer if none of those were imported actually
If this works... good!